libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/postprocessor/RoomBeginningPostProcessor.kt

```diff
@@ -39,8 +36,8 @@ class RoomBeginningPostProcessor(private val mode: Timeline.Mode) {
     }
 
     private fun processForRoom(items: List<MatrixTimelineItem>): List<MatrixTimelineItem> {
-        val roomBeginningItem = createRoomBeginningItem()
-        return listOf(roomBeginningItem) + items
+        // No changes needed, timeline start item is already added by the SDK
+        return items
```

Comment:
DMs will now have the new item added by the SDK, it was not the case before. Now we can see "This is the beginning of..." for DM too. To be confirmed by product?

<img width="376" alt="image" src="https://github.com/user-attachments/assets/698b1901-a4a7-4381-ab6f-3b5ed2902525" />

Reply:
I can remove it for now maybe, so it follows the previous behaviour.

Reply:
Latest changes tested OK, thanks!